packages/css/src/identifier.test.ts

```diff
@@ -3,26 +3,58 @@ import { generateIdentifier } from './identifier';
 
 describe('identifier', () => {
   beforeAll(() => {
-    setFileScope('test');
+    setFileScope('path/to/file.css.ts');
   });
 
   afterAll(() => {
     endFileScope();
   });
 
-  it(`should create a valid identifier`, () => {
+  it(`should ignore file scopes without a file extension when creating a path prefix`, () => {
+    setFileScope('test');
     expect(generateIdentifier(undefined)).toMatchInlineSnapshot(`"skkcyc0"`);
```

Comment:
This test is mostly here just to show that the logic is the same as before, we just couldn't see the usual output from this function because the file scope was invalid (`test` instead of `test.css.ts` or something similar).